packages/components/descriptions/src/description-item.ts

```diff
@@ -3,7 +3,7 @@ import { buildProps } from '@element-plus/utils'
 
 import type { ExtractPropTypes, Slot, VNode } from 'vue'
 
-const descriptionItemProps = buildProps({
+export const descriptionItemProps = buildProps({
   /**
```

Comment:
Maybe you should add `export * from './src/description-item'` in `index.ts`, otherwise it seems that you still can’t get it.

https://github.com/element-plus/element-plus/blob/546b21ea8253f463ee00691230063edb77390ea0/packages/components/descriptions/index.ts#L14

Reply:
已添加 @btea